app/src/main/java/io/sam43/gitfolio/utils/Extensions.kt

```
@@ -17,16 +24,30 @@ fun Int.toFormattedCountString(includeDecimalForThousands: Boolean = false): Str
                 "${this / 1000}K"
             }
         }
+
         this < 1_000_000_000 -> { // Millions
             String.format("%.1fM", this / 1_000_000.0).replace(".0M", "M")
```

detekt (1.23.8)

[warning] String.format("%.1fM", this / 1_000_000.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)